src/v0/destinations/google_adwords_offline_conversions/data/TrackAddStoreConversionsConfig.json

```diff
@@ -17,7 +17,8 @@
     ],
     "required": true,
     "metadata": {
-      "type": "toNumber"
+      "type": "toNumber",
+      "regex": "^([1-9]\\d*(\\.\\d+)?|0\\.\\d+)$"
```

Comment:
this first validates the string and then applies toNumber, can we implicitly check this based on toNumber

Reply:
Didn't get that. According to the implementation, we are applying `toNumber` function to convert the value into a function and then using the regex to validate if that is greater than 0 or not.

Reply:
Instead of using regexes for these, as they are not self-explanatory, it is better to create special conditions and write validation logic based on these conditional keys.
Example:
`conditions: ["positive_number"]`

Reply:
I thought adding regex as metadata would add an extra feature. We can use this if any use case needs regex validation for any field.

Reply:
you are right but regex is not easy for all cases.

Reply:
if this is blocking then we can take this later as well